doc/newsboat.asciidoc

```diff
@@ -1038,7 +1038,7 @@ sequence identifier `[char]` contains any non-whitespace characters, then
 `[format 1]` will be evaluated and inserted, otherwise `[format 2]` will be
 evaluated and inserted. The `&` and `[format 2]` are optional, i.e. if the
 identifier's text is empty, then an empty string will be inserted.
-Newsboat also provides a way center a string in a given width and with any given whitespace character with `%=[char][width]`. If width it = 0 the whole width of the window will be used.
+Newsboat also provides a way to center a string in a given width followed by the actual sequence identifier, which is usually a single letter. It is done through `%=[width][identifier]`. If width it = 0 the whole width of the window will be used.
```

Comment:
It actually makes sense to move that to the previous paragraph. You can then avoid mentioning that a sequence identifier is a single letter: just show the format, say it centers the value, and mention the "width=0" corner case.

By the way: sorry I thought of that so late, but can you please add a test for that corner case?

Reply:
I don't exactly how to manage that test, when the width is then dynamically defined by the window with and I therefore cannot compare to a reference string.

Reply:
Tests call `do_format()`, the second argument of which is the window width. That value travels through the calls and ends up in the `width` in `format_format()`. So just pass something other than zero as a second argument in a test.

Reply:
I added the corresponding test and fixed the documentation.